File: app/src/warsh/java/com/quran/labs/androidquran/data/QuranFileConstants.java
package com.quran.labs.androidquran.data;

import com.quran.labs.androidquran.database.DatabaseHandler;
import com.quran.labs.androidquran.ui.util.TypefaceManager;

public class QuranFileConstants {
  // server urls
  public static final int FONT_TYPE = TypefaceManager.TYPE_UTHMANIC_WARSH;

  // arabic database
  public static final String ARABIC_DATABASE = "quran.ar.warsh.db";
  public static final String ARABIC_SHARE_TABLE = DatabaseHandler.ARABIC_TEXT_TABLE;
  public static final boolean ARABIC_SHARE_TEXT_HAS_BASMALLAH = true;
  public static final boolean FETCH_QUARTER_NAMES_FROM_DATABASE = true;
}

File: app/src/warsh/java/com/quran/labs/androidquran/data/QuranFileConstants.kt
package com.quran.labs.androidquran.data

import com.quran.labs.androidquran.ui.util.TypefaceManager
import com.quran.labs.androidquran.database.DatabaseHandler

object QuranFileConstants {
  // server urls
  const val FONT_TYPE = TypefaceManager.TYPE_UTHMANIC_WARSH

  // arabic database
  const val ARABIC_DATABASE = "quran.ar.warsh.db"
  const val ARABIC_SHARE_TABLE = DatabaseHandler.ARABIC_TEXT_TABLE
  const val ARABIC_SHARE_TEXT_HAS_BASMALLAH = true
  const val FETCH_QUARTER_NAMES_FROM_DATABASE = true

  const val FALLBACK_PAGE_TYPE = "warsh"

  var ICON_RESOURCE_ID = com.quran.labs.androidquran.pages.warsh.R.drawable.icon
}
